src/bielewicz-jacek/Fsd.Jacek.Cs/Fsd.Jacek.Cs.Ex4/Views/Product/Forms/Index.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using Fsd.Jacek.Cs.Ex4.Data;
+@using Fsd.Jacek.Cs.Ex4.Models;
+@model ProductModel
+
+@{
+    IDictionary<ProductType, string> productViews = new Dictionary<ProductType, string>
+        {
+        {ProductType.Alkohol, "Alkohol" },
```

Comment:
nie jest dobrym pomysłem definiować w widoku liste dostępnych typów produktu - co jezeli ktoś w bazie doda nowy typ? nigdy nie znajdziesz tego miejsca. To powinno być pobierane przez serwis i wstawiane do modelu widoku